sensorpush_to_statusdb.py

```diff
@@ -0,0 +1,325 @@
+"""Fetch sample readings from sensorpush API, summarize out-of-limit periods
+   and upload those to StatusDB.
+
+   Seems to be one reading per minute given by the API, so nr of samples can be
+   seen as the number of minute fetched, for example 1440 samples for 24h.
+"""
+
+import requests
+import argparse
+import yaml
+import os
+import pytz
+import datetime
+import numpy as np
+import pandas as pd
+import logging
+from couchdb import Server
+
+
+
+class SensorPushConnection(object):
+    def __init__(self, email, password):
+        self.email = email
+        self.password = password
+        self._authorized = False
+        self.base_url = 'https://api.sensorpush.com/api/v1/'
+        self.access_token = None
+
+    def _authorize(self):
+        url_ending = 'oauth/authorize'
+        url = '/'.join(x.strip('/') for x in [self.base_url, url_ending] if x)
```

Comment:
Why not leave out the ending `/` in the base url so you don't have to strip it out later, if I'm right in assuming that's what is done here?

Reply:
Good point, yeah I guess that would work...

Reply:
Adjusted!